### PR TITLE
J hardaway add google analytics back

### PR DESCRIPTION
**Pull request by jeromehardaway**
During the migration, google analytics was not added to the code. If you open the dev tools, go to the console and type `dataLayer` you will get a read out to let you know it exists, which means it is working.

### CODE REVIEW

**Review by Cameron-Porter — APPROVED** [review]

File: src/pages/_app.js [hunk 1]
```diff
@@ -57,6 +58,21 @@ export default function App({ Component, pageProps }) {
           <Footer />
         </main>
       </ThemeProvider>
+      <Script
+        strategy="lazyOnload"
```
Comment (andrew-t-james) [inline src/pages/_app.js:62]: Please change to `strategy="afterInteractive"`.

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: .env.local [hunk 2]
```diff
@@ -23,3 +23,4 @@ MENTOR_WEBHOOK_ID=
 ZIPCODE_GETTER_USERID=
 JOBSEARCH_API_ID=
 JOBSEARCH_API_KEY=
+NEXT_PUBLIC_GOOGLE_ANALYTICS=YOUR_GOOGLE_ANALYTICS_ID=UA-73856435-2
```
Comment (gixxerblade) [inline .env.local:26]: Is this correct?

File: src/pages/_app.js (hunk 1 above)
Comment (jeromehardaway) [reply src/pages/_app.js:62]: Wilco

File: .env.local (hunk 2 above)
Comment (jeromehardaway) [reply .env.local:26]: Our env.local shout be in our gitignore. I will check it out to make sure its fixed.

Comment (jeromehardaway) [reply .env.local:26]: Should be fixed now

File: src/pages/_app.js (hunk 1 above)
Comment (jeromehardaway) [reply src/pages/_app.js]: Did

**Review by gixxerblade — APPROVED** [review]

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: src/pages/_app.js [hunk 3]
```diff
@@ -57,6 +58,21 @@ export default function App({ Component, pageProps }) {
...
+        src={`https://www.googletagmanager.com/gtag/js?id=${process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS}`}
+      />
+
+      <Script id="google-analytics" strategy="lazyOnload">
```
[the remark this answers is not in the document]
Comment (andrew-t-james) [reply src/pages/_app.js:66]: Please fix strategy her as well.

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

**Pull request merged**